index.html

```diff
@@ -12,8 +12,64 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
-<body>
-  <h1>My Resume</h1>
-</body>
+<div class="container">
+  <body>
+    <header>
+      <h1>Isaac Middleton - Web Dev</h1>
+      <nav>
+          <a href="#">Home</a>
+          <a href="#">About</a>
+          <a href="#">Portfolio</a>
+          <a href="#">References</a>
+      </nav>
+    </header>
+      <img src="images/pic06.jpg" alt="#" /> 
+    <div class="main-content">
+      <section class="intro">
+        <h2>About Me</h2>
+        <p>I am a student of Web Development. I have finished a few projects and am working on more! 
+          I am a hard worker and great with people.</p>
+      </section>
+      <section class="skills">
```

Comment:
This is purely preference, but some developers like to have an empty line separating each section, header, footer, ect. It just makes it a bit easier to read.